.gitignore

```diff
@@ -44,5 +44,6 @@ profile
 *.suo
 
 .vs
-out
```

Comment:
Leave it as it was, because it's the default output directory used by Visual Studio for CMake

Reply:
This was a duplicate, it's already defined on line 3. When removing this line, I checked and the dir was still being ignored

Reply:
woops lol